package.json

```diff
@@ -22,19 +22,20 @@
   },
   "homepage": "https://github.com/poanetwork/posdao-contracts#readme",
   "dependencies": {
-    "solidity-docgen": "^0.1.1",
+    "solidity-docgen": "^0.2.0-alpha.1",
```

Comment:
Let's return the `0.1.1` version because as far as I know, `0.2.0` completely differs from `0.1.1`, so the `npm run docs-start` and `npm run docs-publish` commands won't work.

Reply:
I've tested. Generated docs: https://maxaleks.github.io/posdao-contracts/
`0.1.1` hadn't worked for me. There was an error related to `solc`
`0.2.0` version supports `solc-js`

Reply:
You can see the build details here: https://travis-ci.com/maxaleks/posdao-contracts/builds/116242771

Reply:
I don't see the left menu in your version https://maxaleks.github.io/posdao-contracts/docs/

The correct variant is https://poanetwork.github.io/posdao-contracts/docs/

Reply:
You are right, I haven't checked your docs. Looking at another solution